lesson05/օbject.js

```diff
@@ -0,0 +1,50 @@
+module.exports = {
+   arrToObject,
+   returnHeadAndTail,
+   zipObject
+}
+//exercises 1. Write function that will do reverse operation for
+
+function arrToObject(arr) {
+   const newObj = {};
+   for(let i = 0; i < arr.length; i++) {
+      newObj[arr[i][0]] = arr[i][1]
+   }
+
+   // return newObj = Object.fromEntries(arr);
+   return newObj
+}
+
+// Exercises 2. Write functions that willreturn head and tail of Array
+function returnHeadAndTail(getHead, getTail){
```

Comment:
read problem requirements again, 
separate the logic in different functions,
what this lines means?
`getHead = [1, 2, 3]`
`getTail = [1, 2, 3]`